app/components/LeftPaneEntries/EntryItem/index.tsx

```diff
@@ -177,9 +187,8 @@ function EntryItem(props: EntryItemProps) {
                         <ExcerptInput
                             value={excerpt}
                             image={entryImage}
-                            imageRaw={imageRaw}
-                            // FIXME: pass this after image drag/drop is implemented
-                            leadImageUrl={undefined}
+                            imageRaw={imageRaw ?? leadAttachment?.filePreview?.url ?? ''}
```

Comment:
`ExcerptInput` accepts `imageRaw` to be `undefined`. So, let's
```suggestion
                            imageRaw={imageRaw ?? leadAttachment?.filePreview?.url ?? undefined}
```